src/ui/package.json

```diff
@@ -10,35 +10,38 @@
     "test-ci": "ng test --single-run",
     "pree2e": "webdriver-manager update",
     "e2e": "protractor",
-    "compile": "ng build --prod --progress=false -o rootfs/dist",
-    "compile-aot": "ng build --prod --aot --progress=false -o rootfs/dist"
+    "compile": "ng build --prod --progress=false --output-path rootfs/dist",
+    "compile-aot": "ng build --prod --aot --progress=false --output-path rootfs/dist"
   },
   "private": true,
   "dependencies": {
-    "@angular/common": "2.4.3",
-    "@angular/compiler": "2.4.3",
-    "@angular/core": "2.4.3",
-    "@angular/forms": "2.4.3",
-    "@angular/http": "2.4.3",
+    "@angular/animations": "^4.0.0",
```

Comment:
It may sounds huge, but they actually change from 2 to 4 for "semantic versioning", and the changes are not that important. Plus, it's "backwards compatible with 2.x.x" for most cases.
source: https://angularjs.blogspot.it/2017/03/angular-400-now-available.html